pymeasure/display/widgets/plot_widget.py

```diff
@@ -41,20 +41,27 @@ class PlotWidget(TabWidget, QtWidgets.QWidget):
     """
 
     def __init__(self, name, columns, x_axis=None, y_axis=None, refresh_time=0.2,
-                 check_status=True, linewidth=1, parent=None):
+                 check_status=True, linewidth=1, parent=None, setup=True):
```

Comment:
I must admit that I don't completely understand the changes in this file: if I understand it correctly, the PlotWidget is never called with `setup=False`, is it? So I would think that the changes in this file are not necessary.

Reply:
Reverted this implementation